### PR TITLE
Initial choose instituion and profile setup

### PR DESCRIPTION
**Pull request by andreldsa**
Added chosse institution when user first login, then gives the user the option to setup profile informations.

In addition, changed the way as the authentication is required adding user to the sessionStorage (with bugs).

### CODE REVIEW

File: app/auth/configProfileController.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    var app = angular.module("app");
+    
+    app.controller("ConfigProfileController", function ConfigProfileController($state, InstitutionService, AuthService, UserService) {
+        var configPrileCtrl = this;
+
+        configPrileCtrl.user = AuthService.getCurrentUser();
+
+        configPrileCtrl.finish = function finish() {
+            UserService.save(configPrileCtrl.user).then(function success(info) {
+                console.log(info);
```
Comment (mayzabeel) [inline app/auth/configProfileController.js]: esse console.log é pra ficar mesmo?

File: app/auth/loginController.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+'use strict';
```
Comment (mayzabeel) [inline app/auth/loginController.js]: AInda falta terminar esse, ou não vai usar?

File: app/auth/user.js [hunk 3]
```diff
@@ -38,21 +41,28 @@ User.prototype.isMember = function isMember(institutionKey){
       return _.includes(_.map(this.institutions, getKeyObj), institutionKey);
 };
 
+User.prototype.addInstitution = function addInstitution(institutionKey){
+    this.institutions.push(institutionKey);
+};
+
+User.prototype.getPatch = function getPatch(newUser) {
```
Comment (mayzabeel) [inline app/auth/user.js]: usa onde isso?

File: app/main/mainController.js [hunk 4]
```diff
@@ -38,6 +34,20 @@
             enabled: true
         }, ];
 
+        $rootScope.$on("authorized", function() {
+            console.log(mainCtrl.user);
```
Comment (mayzabeel) [inline app/main/mainController.js]: remover

File: app/main/mainController.js [hunk 5]
```diff
@@ -38,6 +34,20 @@
...
+            console.log(">>>>>>>>>>>>>> authorized: ",mainCtrl.user.institutions);
```
Comment (mayzabeel) [inline app/main/mainController.js]: remover

File: handlers/base_handler.py [hunk 6]
```diff
@@ -11,11 +11,12 @@
...
 
-    @json_response
-    def handle_exception(self, exception, debug):
-        """Handle exception."""
-        logging.error(str(exception))
-        self.response.set_status(500)
-        self.response.write(json.dumps({
-            "msg": "Error! %s" % str(exception)
-        }))
+    pass
```
Comment (mayzabeel) [inline handlers/base_handler.py]: pq precisou ignorar?

**Comment by andreldsa** [comment]
Choice of institution for new users #90

**Comment by andreldsa** [comment]
Pull request quebrado em outros 3 pull requests.

**Pull request closed without merging**